adapters/jquery.js

```diff
@@ -265,10 +265,11 @@
 				} else {
 					// Editor is already during creation process, bind our code to the event.
 					editor.once( 'instanceReady', function() {
-						setTimeout( function() {
+						setTimeout( function integrateWithjQuery() {
```

Comment:
I think it should be more like `waitForEditor` or `waitForEditorElement` since this function just checks if `editor.element` is set.